pywinauto/controls/HwndWrapper.py

```diff
@@ -1224,48 +1224,45 @@ def scroll(self, direction, amount, count = 1, retry_interval = None):
     Scroll = scroll
 
    #-----------------------------------------------------------
-    def get_scroll_info(self, hwnd, fnBar, lpsi):
-        win32functions.GetScrollInfo(hwnd, fnBar, lpsi)
-        if not lpsi:
+    def get_scroll_info(self, scr_bar_type):
+        """Get the scroll bar info"""
+        scr_info = win32structures.LPSCROLLINFO()
+        scr_info.cbSize = ctypes.sizeof(scr_info)
+        scr_info.fMask = win32defines.SIF_PAGE
+        win32functions.GetScrollInfo(self.handle, scr_bar_type, scr_info)
+        if not scr_info:
             return False
         else:
```

Comment:
Return values are both about successful or unsuccessful calls. This is not an info retrieved. We should raise an exception if scroll bar is not supported by the control. Otherwise return some human readable info, not a raw structure.